test-suite/js/src/test/scala/org/scalajs/testsuite/library/LinkingInfoTest.scala

```diff
@@ -26,6 +27,19 @@ class LinkingInfoTest {
   @Test def developmentMode(): Unit =
     assertEquals(!Platform.isInProductionMode, LinkingInfo.developmentMode)
 
+  @Test def assumedESVersion(): Unit =
+    assertEquals(Platform.assumedESVersion, LinkingInfo.assumedESVersion)
+
   @Test def assumingES6(): Unit =
     assertEquals(Platform.assumeES2015, LinkingInfo.assumingES6)
+
+  @Test def esVersionConstants(): Unit = {
+    assertEquals(5, ESVersion.ES5_1)
+    assertEquals(6, ESVersion.ES2015)
+    assertEquals(7, ESVersion.ES2016)
+    assertEquals(8, ESVersion.ES2017)
+    assertEquals(9, ESVersion.ES2018)
+    assertEquals(10, ESVersion.ES2019)
+    assertEquals(11, ESVersion.ES2020)
+  }
```

Comment:
What is the point of this test? It just tests the definitions. IMO a more useful test would be that the values in linker and the one in library agree. But `assumedESVersion` does this implicitly (if we run it for all versions).

Reply:
Since the numeric values are meaningful on their own, I thought they should be tested. I added a comment saying that.